fact/io.py

```diff
@@ -24,12 +26,15 @@
 native_byteorder = native_byteorder = {'little': '<', 'big': '>'}[sys.byteorder]
 
 
-def write_data(df, file_path, key='table'):
+def write_data(df, file_path, key='table', hdf_format='pandas'):
```

Comment:
can we you 'data' instead of 'table' as default key? I think its a little less general.